shenyuClientRust/http.md

```diff
@@ -0,0 +1,158 @@
+---
+description: http
+title: Http Example
+---
+# The Http type Register
+
+**1.Fist make sure The ShenYuAdmin is Started, and ShenYuAdmin service active port is 9095.**
+
+```log
+//Or you will see this error :
+ 
+2024-10-09T07:23:26.970200Z ERROR shenyu_client_rust::core: Can't get register token
+2024-10-09T07:23:29.015550Z ERROR shenyu_client_rust::core: [ERROR], register metadata to http://127.0.0.1:9095/shenyu-client/register-metadata failed, app_name: shenyu_client_app, path: /health, contextPath: /xxx
+2024-10-09T07:23:31.039558Z ERROR shenyu_client_rust::core: [ERROR], register metadata to http://127.0.0.1:9095/shenyu-client/register-metadata failed, app_name: shenyu_client_app, path: /users, contextPath: /xxx
+2024-10-09T07:23:33.089412Z ERROR shenyu_client_rust::core: [ERROR], register uri to http://127.0.0.1:9095/shenyu-client/register-uri failed, app_name: shenyu_client_app, host: 10.10.9.198, port: 3000
+2024-10-09T07:23:35.124249Z ERROR shenyu_client_rust::core: [ERROR], register discover config to http://127.0.0.1:9095/shenyu-client/register-discoveryConfig failed, discovery_type: zookeeper, host: 10.10.9.198, port: 3000
+ 
+```
+
+**2.Step 1 Get shenyu_admin_client. (Register service need this)**
+
+```yml
+#  a yml-format config file
+shenyu:
+  register:
+    register_type: "http"
+    servers: "http://127.0.0.1:9095"
+    props:
+      username: "admin"
+      password: "123456"
+```
+
+```markdown
```

Comment:
This can be ```rust

Reply:
There is a red syntax warning in the IDE if you change to rusty code mode. It's need or not?